components/tools/OmeroPy/src/omero/util/populate_metadata.py

```diff
@@ -892,10 +892,29 @@ def write_to_omero(self):
                 "Unable to create table: %s" % name)
         original_file = table.getOriginalFile()
         log.info('Created new table OriginalFile:%d' % original_file.id.val)
+
+        columns = self.columns
```

Comment:
Minor nitpicky point, there's a mix of `columns` and `self.columns` in the following code, it confused me initially (thought they were two separate vars).